examples/nextjs-lit/README.md

```diff
@@ -0,0 +1,8 @@
+# Lit in Next.js example
```

Comment:
Do we want to name this folder `nextjs-lit` or just `nextjs`? We might end up having `-lit` on a lot of things.

Reply:
done